.github/workflows/ci.yaml

```diff
@@ -85,4 +110,5 @@ jobs:
           draft: "true"
           files: |-
             _out/siderolink-agent-*
+            _out/siderolink-client-*
```

Comment:
do we need binaries to be published?

Reply:
I guess we also need to disable pushing these container images

Reply:
Moved to the `hack` directory.

Reply:
i guess needs a rekres?

Reply:
Nope. `kres` ignores hack directory.